dartsim/src/EntityManagementFeatures.cc

```diff
@@ -632,7 +732,31 @@ Identity EntityManagementFeatures::ConstructEmptyModel(
         dart::dynamics::Frame::World(),
         _name + "_frame");
 
-  auto [modelID, modelInfo] = this->AddModel({model, modelFrame, ""}, _worldID); // NOLINT
+  auto [modelID, modelInfo] =
+      this->AddModel({model, _name, modelFrame, ""}, _worldID);  // NOLINT
+
+  return this->GenerateIdentity(modelID, this->models.at(modelID));
+}
+
+/////////////////////////////////////////////////
+Identity EntityManagementFeatures::ConstructEmptyNestedModel(
+    const Identity &_modelID, const std::string &_name)
```

Comment:
I'd suggest renaming `_modelID` to `_parentModelID` to avoid confusion with `modelID` below.

Reply:
https://github.com/ignitionrobotics/ign-physics/pull/230/commits/62abc8babaa68db286f0ab24657042f83c7504c6